File: Selenium/Activities/Activity12.java
package activities;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity12 {
    public static void main(String[] args){
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get(" https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("Title is :" + driver.getTitle());
        WebElement text_field = driver.findElement(By.id("input-text"));
        System.out.println("Is Text field Enabled :"+text_field.isEnabled());
        WebElement Enable_Button =driver.findElement(By.id("toggleInput"));
        Enable_Button.click();
        System.out.println("Is text field Enabled :" +text_field.isEnabled());
        driver.close();

    }
}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("Title is :" + driver.title)
    text_field = driver.find_element(By.ID,"input-text")
    print("Is Text field Enabled :",text_field.is_enabled())
    driver.find_element(By.ID,"toggleInput").click()
    print("Is Text field Enabled :",text_field.is_enabled())
    text_field.send_keys("I am Enabled")
    print("input Text is :",text_field.get_attribute('value'))
    driver.close()




